roadmap/implementers-guide/src/runtime/disputes.md

```diff
@@ -66,6 +65,7 @@ PostConclusionAcceptancePeriod: BlockNumber;
   1. Fail if any disputes in the set are duplicate or concluded before the `PostConclusionAcceptancePeriod` window relative to now.
   1. Pass on each dispute statement set to `provide_dispute_data`, propagating failure.
   1. Return a list of all candidates who just had disputes initiated.
+
 * `provide_dispute_data(DisputeStatementSet) -> bool`: Provide data to an ongoing dispute or initiate a dispute.
   1. All statements must be issued under the correct session for the correct candidate. 
```

Comment:
We'll eventually need some framework for testing ousted validators acting in their now-ended session where they were still a validator.  Ugh.

Reply:
Oh you mean validators who have been silenced? Good point...